source/extensions/filters/common/local_ratelimit/local_ratelimit_impl.h

```diff
@@ -36,9 +36,11 @@ class LocalRateLimiterImpl {
     mutable std::atomic<uint32_t> tokens_;
     MonotonicTime fill_time_;
   };
+  int64_t counter_{0};
   struct LocalDescriptorImpl : public RateLimit::LocalDescriptor {
     std::shared_ptr<TokenState> token_state_;
     RateLimit::TokenBucket token_bucket_;
+    int64_t multiplier_;
```

Comment:
It probably doesn't matter given the modulo arithmetic, but should these be unsigned for clarity? In general could you add more comments around these variables? It's not immediately clear what this stuff is for.

Reply:
Yeah, `uint64_t` would be better here. Sorry for my negligence in the reivew 😞.